src/Eurofurence.App.Server.Web/Controllers/UsersController.cs

```diff
@@ -33,5 +44,38 @@ public UserRecord GetUsersSelf()
             }
             return result;
         }
+
+        /// <summary>
+        /// Sets (or unsets) a penalty for the artist alley for a given user
+        /// </summary>
+        /// <param name="id">The ID of the user</param>
```

Comment:
Clarify which ID is to be used here (should be the user ID from the identity provider), so renaming the field to "identityId" might also help for clarification here.